tests/ProphecyTraitTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Prophecy\PhpUnit\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+
+class ProphecyTraitTest extends TestCase
+{
+    use ProphecyTrait;
+
+    public function testProphesizeWithoutArguments(): void
+    {
+        $prophecy = $this->prophesize()->reveal();
```

Comment:
I added this test in 49b644488, because I realized, while addressing [your comment](https://github.com/phpspec/prophecy-phpunit/pull/45#discussion_r1158546387), that passing no class made the trait fatal. 

This is basically a regression test, and it doesn't have to fail.

Reply:
I understand why you added a regression test for that case. but what you added is not a regression test of the trait but a usage of the trait (as that trait runs in the main phpunit process).

If the test regresses, we would like to have a failed (or errorred) tests, not a fatal for the whole testsuite. See all the other existing testcases.

Testing an extension of the testing tool requires a dedicated way to write tests.

Reply:
Understood! Done in a8c51be